test/tests/kernels/functor_kernel/functor_kernel.i

```diff
@@ -0,0 +1,83 @@
+# Heat conduction with fixed temperature on left and convection BC on right.
+# Source is temperature-dependent:
+#   S(T)    = B   - A   * (T - T_inf)^2 [W]
+#   S'''(T) = B/V - A/V * (T - T_inf)^2 [W/m^3]
+# Assume volume V = 1 m^3, so
+#   S'''(T) = B   - A   * (T - T_inf)^2 [W/m^3]
+
+k = 15.0
+htc = 100.0
+T_ambient = 300.0
+source_coef_A = 0.1
+source_coef_B = 1e4
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+[]
```

Comment:
This is completely random, but now that I'm slowly working on enabling more proper unit testing...

Is there a reason why we just naturally use 10x10 grids for tests? What does 2D get us here that a few elems in 1D won't?